akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -1,154 +1,51 @@
-# Cluster Usage
-
-For introduction to the Akka Cluster concepts please see @ref:[Cluster Specification](common/cluster.md).
+# Classic Cluster Usage
+ 
+This document describes how to use Akka Cluster and the Cluster APIs using code samples.
+For specific documentation topics see:  
 
-The core of Akka Cluster is the cluster membership, to keep track of what nodes are part of the cluster and
-their health. There are several @ref:[Higher level Cluster tools](cluster-usage.md#higher-level-cluster-tools) that are built
-on top of the cluster membership.
+* @ref:[Cluster Specification](typed/cluster-specification.md)
+* @ref:[User Guide](typed/cluster.md)
```

Comment:
shouldn't we have the Classic vs new API include here somewhere, and then this User Guide link should be below that

Reply:
If the user is reading this they are already on the Classic API page and just need to read on.  Though we can be more explicit.